dd-sdk-android/src/main/kotlin/com/datadog/android/tracing/internal/data/TraceWriter.kt

```diff
@@ -30,4 +39,54 @@ internal class TraceWriter(
     override fun incrementTraceCount() {
         // NO - OP
     }
+
+    // endregion
+
+    // region Internals
+
+    private fun sendRumErrorEvent(span: DDSpan) {
+        (GlobalRum.get() as? AdvancedRumMonitor)?.addErrorWithStacktrace(
+            spanErrorMessage(span),
+            RumErrorSource.SOURCE,
+            span.tags[DDTags.ERROR_STACK]?.toString(),
```

Comment:
On iOS we consider errors coming from Logger and Tracer as `RumErrorSource.LOGGER`, as user doesn't send them directly from the code. Maybe we should use `.SOURCE` if the stack is available, and `LOGGER` otherwise? WDYT?

Reply:
Hmm...that would be an isolated case for us, not sure what to say. WDYT @xgouchet

Reply:
Also @ncreated if you decide to use LOGGER we should use it in both cases I think as the User can log the error message without the Stack and the other way around. For me it is a LOG in both cases.

Reply:
In this particular case, spans are marked as error manually, so I'd consider this as SOURCE, and we only use LOGGER for errors copied from the Logger.